data/sql/updates/pending_db_world/rev_1729968739299017200.sql

```diff
@@ -0,0 +1,45 @@
+INSERT INTO `spell_script_names` (`spell_id`, `ScriptName`) VALUES (51000, 'spell_hos_dark_matter_size');
```

Comment:
Missing DELETE statement before INSERT.
Also the name does not match the script name in the source code, which is rhw reason for CI failing.

Reply:
Ok, I corrected.